src/main/java/br/com/caelum/clines/api/locations/LocationView.java

```diff
@@ -1,10 +1,12 @@
-package br.com.caelum.clines.api.airports;
+package br.com.caelum.clines.api.locations;
```

Comment:
This class was not used in airport package anymore?

Reply:
It is, but it's used at location, and we considered it is the right place to it be.

Reply:
I understand, but I believe that LocationView is a DTO that would only have properties that are necessary for the airport context, so I think it makes sense to duplicate this class in different packages. LocationViewMapper enters the same case.

Reply:
@andrelbravo I don`t think code duplication is a good approach.

Reply:
@andrelbravo , fixed